### ISSUE TITLE
backup rotation script: change to pure-python in a thread

### ISSUE THREAD

**Issue by caver456** [ncssar/radiolog#602]
Right now, the backup rotation script is a powershell script.

1) this would only work on Windows, which would be a problem if/when radiolog is to be made usable by macOS and/or linux
2) this has caused various errors and issues over time.  The latest is #601 - failure to find '.\rotateCsvBackups.ps1' in 3.4.0 - which is a simple relative-path error due to not running the executable while the current directory is the install directory.

The main reason for making it a powershell script in the first place was the ability to background it, so that the rotation would not cause lag - see #364 and [the commit that fixed it.](https://github.com/ncssar/radiolog/commit/b013955fa06530e38d103df4ad5dcab334b256d9)

Hopefully we can make it pure-python in a background thread.  This would be the first foray into threading for radiolog, but, sartopo_python uses threading successfully, and it is getting incorporated in other current issues - see #600 - so maybe switching over won't be that hard.

**Comment by caver456** [comment]
Another issue is that the rotation script output doesn't get saved in the transcript file.  It's probably possible to address that separately, but, there's currently no real need to do so.

**Comment by caver456** [comment]
Closing as fixed - not sure what the pending-fixed was still waiting on - it's working in 3.99.0
